src/steps/measureStep.ts

```diff
@@ -12,16 +12,13 @@ export const measureStep = (
   const stepMetricName = S + step;
   const startMarkExists = WP.getEntriesByName(M + startMark).length > 0;
   const endMarkExists = WP.getEntriesByName(M + endMark).length > 0;
-  if (!endMarkExists || !config.steps) {
+  if (!endMarkExists || !startMarkExists || !config.steps || !config.steps[step]) {
     return;
   }
 
   const { maxOutlierThreshold, vitalsThresholds } =
     STEP_THRESHOLDS[config.steps[step].threshold];
 
-  if (!startMarkExists) {
-    return;
-  }
```

Comment:
combined this check into the one above

Reply:
is `!config.steps[step]` another scenario when we should exit this?

Reply:
yup its in there 
![image](https://github.com/Zizzamia/perfume.js/assets/50121662/d0f6b844-de87-46aa-af6f-f1ae83e769bb)

Reply:
I challenge both of you to think outside the box, and yes the code has to work correclty BUT also has to be the most efficient as possible.

Any coder can write code that works, but only the best one can write code that is the most **efficient**, **elegant**, **easy to mantain**.... and also **works**.

@Cdub-9 when a colleagues mention something over code review, I advice you to challenge the code not the person, and use sentence like "yeah we have that part covered, althought I think how we can make this function even better, any thoughts @lkuangPR " this will be a better way to work together.

Just protecting the code you wrote does not make the code better, it just delays than another eng ("in this case me") comes and read the code and find 3~4 different ways to improve it.

Use Perfume.js to challenge yourself in writing the best code of your life. I know you two are capable of incredible code. ✨